lib/inspec/plugins/resource.rb

```diff
@@ -28,6 +28,12 @@ def desc(description = nil)
       __resource_registry[@name].desc(description)
     end
 
+    def supports(criteria = nil)
+      return if criteria.nil?
+      Inspec::Resource.supports[@name] ||= []
+      Inspec::Resource.supports[@name].push criteria
```

Comment:
Can you wrap the parameter in parentheses so we remain consistent in the codebase?

```ruby
Inspec::Resource.supports[@name].push(criteria)
```